test/box-luatest/gh_9186_7331_forbid_db_access_and_txn_control_in_txn_triggers_test.lua

```diff
@@ -12,7 +12,7 @@ g.after_each(function(cg)
     cg.server:drop()
 end)
 
-g.test_database_access_in_txn_triggers = function(cg)
```

Comment:
For what it's worth, maybe just squash the changes?

Reply:
It seems to me that these 2 commits are highly independent, so I wouldn't squash the changes.
However, I add changelog entry as a separate commit, if it feels better.

Reply:
Done.